products/sqlite.md

```diff
@@ -11,6 +11,13 @@ changelogTemplate: "https://www.sqlite.org/changes.html#version_{{'__LATEST__'|r
 releaseDateColumn: true
 eolColumn: Support Status
 
+identifiers:
+-   purl:  pkg:generic/sqlite
+-   purl:  pkg:apk/alpine/sqlite
+-   purl:  pkg:github/sqlite/sqlite
+-   purl:  pkg:rpm/sqlite
```

Comment:
Not a big fan of those OS-less PURL as I don't know how to validate them. Should we remove it ?